package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "name": "mstranslator",
   "description": "Microsoft Translator API module for node.js",
-  "version": "2.0.3",
+  "version": "2.0.5",
```

Comment:
Let the package maintainer take care of this -- it's better to merge to master, *then* bump the version using `npm version patch`. :)